ts/smelter-core/src/api/input.ts

```diff
@@ -13,6 +13,7 @@ import { _smelterInternals } from '@swmansion/smelter';
  */
 export type RegisterInputRequest =
   | Api.RegisterInput
+  | { type: 'mp4_blob'; blob: any }
```

Comment:
`Api.RegisterInput` is a generated type and already contains definitions for `mp4`; hence, the new type `mp4_blob`

Reply:
I don't like adding new type just for that, it might make sense if it was more generic to handle some other cases types in the future, but not in this form